lib/events/food-events.js

```diff
@@ -1,108 +1,72 @@
+const $foodFilter      = $('input[name="food-filter"]');
+const $foodTable       = $('#foods-table');
+const $createFood      = $('#create-food-btn');
+const $calorieHeader   = $('.calories-header');
 
 Food.getAllFood()
   .then((foods) => {
     foodCollection = foods.map((food) => {
-      return new Food(food)
+      return new Food(food);
     }).sort((a,b) => {
-      return b.id - a.id
-    })
-    helper.refreshTable(foodCollection)
-  })
+      return b.id - a.id;
+    });
+    view.refreshTable(foodCollection);
+  });
 
 $submit.on('click', (event) => {
-  if (helper.validateInputs() === true) {
+  if (view.validateInputs() === true) {
     Food.addFood()
       .then((food) => {
-        foodCollection.push(food[1])
-        helper.clearInputs()
-        helper.resetDescriptions()
-        $('#foods-table > tbody > tr').eq(0).after(food[0])
-      })
-  }
-})
+        foodCollection.push(food[1]);
+        view.clearInputs();
+        view.resetDescriptions();
+        $('#foods-table > tbody > tr').eq(0).after(food[0]);
+      });
+  };
+});
 
 $foodFilter.on('keyup', (event) => {
-  let filter = $foodFilter.val().toLowerCase()
+  let filter = $foodFilter.val().toLowerCase();
   let filteredFoods = foodCollection.filter((food) => {
-    return food.name.toLowerCase().includes(filter)
+    return food.name.toLowerCase().includes(filter);
   }).sort((a,b) => {
-    return b.id - a.id
-  })
-  helper.refreshTable(filteredFoods)
-})
+    return b.id - a.id;
+  });
+  view.refreshTable(filteredFoods);
+});
 
 $foodTable.on('click', '.delete-food', function(event) {
-  const itemID = parseInt(this.dataset.id)
+  const itemID = parseInt(this.dataset.id);
   foodCollection = foodCollection.filter((food) => {
-    return food.id !== itemID
-  })
-  helper.refreshTable(foodCollection)
+    return food.id !== itemID;
+  });
+  view.refreshTable(foodCollection);
 
-  Food.delete(itemID)
-})
+  Food.delete(itemID);
+});
 
-$foodTable.on('click', '.food-name', function(event) {
-  const itemID = this.dataset.id
-  this.outerHTML = `<td><input type="text" class="edit-input edit-name"` +
-    ` data-id=${itemID} value="${this.innerText}"></td>`
-})
+$foodTable.on('click', '.food-name', (event) => {
+  view.changeToInput(event, 'name')
+});
 
-$foodTable.on('click', '.food-calories', function(event) {
-  const itemID = this.dataset.id
-  this.outerHTML = `<td><input type="text" class="edit-input edit-calories"` +
-    ` data-id=${itemID} value="${this.innerText}"></td>`
-})
+$foodTable.on('click', '.food-calories', (event) => {
+  view.changeToInput(event, 'calories')
+});
 
-$foodTable.on('mouseout', '.edit-name', function(event) {
-  const itemID = parseInt(this.dataset.id)
-  let food = foodCollection.find((food) => { return food.id === itemID })
-  food.name = this.value
-  food.update()
-  helper.refreshTable(foodCollection)
-})
+$foodTable.on('mouseout', '.edit-name', (event) => {
+  view.acceptEdits(event, 'name');
```

Comment:
one further way to simplify this would be to have the event just be:
```javascript
$foodTable.on('mouseout', '.edit-name', view.acceptEdits);
```
Then `event` gets automatically given to `view.acceptEdits`. One down side is you would need to query a data element from the `event.target` to get `name` or `calories` or use string manipulation to parse that out of the classes (`.edit-name` and `.edit-calories`).

No need for a change, just a thought on a different approach.